### PR TITLE
V1.26.1 stage

### PR DESCRIPTION
**Pull request by yumex93**
### Testing

New tests cover the changes: 

### Licensing

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,4 +1,6 @@
 # Changelog
+## 1.26.1
+* Enhancement - Set up pause container user the same as proxy container when App Mesh enabled
```
Comment (fenxiong) [inline CHANGELOG.md]: Can you add this:
Bug - Fixed a bug where network stats are not presented in container stats https://github.com/aws/amazon-ecs-agent/pull/1932

Comment (petderek) [reply CHANGELOG.md]: Good catch.

**Comment by yumex93** [comment]
TestShutdownOrder timeouts in windows integ test

**Comment by yumex93** [comment]
TestContainerMetadataFile tests fail on linux/windows/arm instances

**Pull request merged**